api/src/main/resources/application-local.properties

```diff
@@ -4,10 +4,9 @@ spring.datasource.url=jdbc:postgresql://localhost:5432/payment
 spring.datasource.username=payment
 spring.datasource.password=payment
 spring.datasource.driver=org.postgresql.Driver
-
 spring.liquibase.enabled=${SPRING_LIQUIBASE_ENABLED:true}
 spring.liquibase.change-log=classpath:/db/changelog/db.changelog-master.xml
-
+server.port=8070
```

Comment:
what is the reason behind this?

Reply:
Removed now, Previously added for local port conflict issue.